.snippets/code/tutorials/tooling/zombienet/overview/base-example.json

```diff
@@ -3,8 +3,6 @@
         "timeout": 1000,
         "bootnode": false,
         "provider": "kubernetes",
-        "backchannel": false,
-        ...
-    },
-    ...
```

Comment:
I don't think the pre-commit hook should remove the `...` since it has been added to showcase that there is content before and after this portion of the JSON file.

The same applies for the other instances